contracts/vaults/BaseVaults/RibbonThetaVault.sol

```diff
@@ -395,14 +406,29 @@ contract RibbonThetaVault is RibbonVault, RibbonThetaVaultStorage {
      * @notice Rolls the vault's funds into a new short position.
      */
     function rollToNextOption() external onlyKeeper nonReentrant {
+        uint256 currQueuedWithdrawShares = currentQueuedWithdrawShares;
+
         (
             address newOption,
             uint256 lockedBalance,
             uint256 queuedWithdrawAmount
-        ) = _rollToNextOption(uint256(lastQueuedWithdrawAmount));
+        ) =
+            _rollToNextOption(
+                lastQueuedWithdrawAmount,
+                currQueuedWithdrawShares
+            );
 
         lastQueuedWithdrawAmount = queuedWithdrawAmount;
 
+        uint256 newQueuedWithdrawShares =
+            uint256(vaultState.queuedWithdrawShares).add(
+                currQueuedWithdrawShares
+            );
+        ShareMath.assertUint128(newQueuedWithdrawShares);
```

Comment:
`currentQueuedWithdrawShares` is uint256 and gets added to `vaultState.queuedWithdrawShares` which is uint128. There can be an overflow when casting the uint256 to uint128.

It wont overflow but `assertUint128` uses very little gas so might as well check for it just in case.